### ISSUE TITLE
idea: forward to mail address ?

### ISSUE THREAD

**Issue by rycks** [nextcloud/ocsms#170]
Hello,
this is just an idea : send sms message to my mail address so i can have informations even if i forgot my phone at home for example ...

thanks,
Éric

**Comment by enoch85** [comment]
@rycks Why, when you have the App in Nextcloud that you can access from anywhere you want?

**Comment by nerzhul** [comment]
i don't understand the interest too, and i don't want to re-invent a client mail, whereas you have the webapp

**Comment by rycks** [comment]
Ok, i'm sorry, just imagine this situation: i forget my phone at home, somebody send me a sms to my phone number (for example my wife "don't forget to get bred" ... i can't have this information ... but if ocsms can forward all sms to a mail addr i could get the information on my email adress (on my computer for example at work) ...

**Comment by nerzhul** [comment]
i see the usecase but it's a big usecase, i don't know if nextcloud has a embedded library to send mails, if it is maybe it can be nice

**Comment by rycks** [comment]
Here is an horrible hack as proof of concept ... and save my life ... an other use case is when i goes abroad, my SIM card stays at home with an old smartphone and ocsms forward all SMS to my email address ... (very good things for two system authenticate : password + sms code, for online payment with my bank who sends me a "secure code by sms" ... and so on)

POC : https://oc.garluche.fr/index.php/s/SwHGDeb1zMhIJfe

**Comment by nerzhul** [comment]
yeah it's a ugly hack to do this in the mapper which is the DB interface, but idea sounds good, maybe i should add this as a new feature heh, push on sync + daily report ?

**Comment by e-alfred** [comment]
How about using the notification system of Nextcloud to send a notification mail "New SMS from ocsms came in, go open your Nextcloud web interface at https://..."

This wouldn't be a reinvention of an existing system but rather using what is already there.

**Comment by rycks** [comment]
Hello again,
here is a new update for this idea https://github.com/rycks/ocsms

Éric

**Comment by nerzhul** [comment]
sounds to be an interesting start for a such thing, i suggest to use nextcloud crons instead of pushing each SMS by mail, or have 2 checklists to select one or both systems

**Comment by 9662** [comment]
> this is just an idea : send sms message to my mail address so i can have informations even if i forgot my phone at home for example ...

Surely there is [at least one phone application in the world](https://www.google.com/search?q=forward+sms+to+email) that does that directly from the phone, without involving a third actor at all?
